compiler/grainlsp/utils.re

```diff
@@ -0,0 +1,814 @@
+open Grain;
+open Compile;
+open Grain_parsing;
+open Grain_utils;
+open Grain_typed;
+open Grain_diagnostics;
+
+let mark_down_grain = code => "```grain\n" ++ code ++ "\n```";
```

Comment:
Should probably call this `markdown_grain` 😛

Reply:
:-D